galley/pkg/server/server.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Istio Authors
+// Copyright 2019 Istio Authors
```

Comment:
I believe copyright date corresponds to when the file was _created_ and not the latest update. cc @geeknoid